packages/js/components/src/experimental-select-control/combo-box.scss

```diff
@@ -7,8 +7,10 @@
 	display: flex;
 	flex-wrap: wrap;
 	align-items: center;
-	padding: $gap-smallest 36px $gap-smallest $gap-smaller;
+	padding: 2px 36px 2px $gap-smaller;
```

Comment:
Was using `2px` still necessary versus `4px` - $gap-smallest after adding the `min-height: 36px`?

Reply:
Yes because min-height doesn't prevent the input field growing in height if its content is greater than 36px. The `[selected items height] + (2 * $gap-smallest)` is greater than 36px.

Reply:
Sounds good, thanks for the clarification :)